phpunit.xml.dist

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0"?>
 <phpunit
     xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
-    xsi:noNamespaceSchemaLocation="vendor/phpunit/phpunit/phpunit.xsd"
+    xsi:noNamespaceSchemaLocation="https://schema.phpunit.de/9.3/phpunit.xsd"
```

Comment:
This should still point to `vendor/phpunit/phpunit/phpunit.xsd`, as it will therefore stick to what's installed (and validate against it in the editor)